meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -329,7 +329,6 @@ IMXBOOT_TARGETS_mx8mm = "flash_evk_spl"
 ## iMX8MM EVK
 OSTREE_KERNEL_ARGS_imx8mmevk ?= "console=tty1 console=ttymxc1,115200 earlycon=ec_imx6q,0x30890000,115200 root=/dev/mmcblk2p2 rootfstype=ext4"
 PREFERRED_PROVIDER_virtual/dtb_imx8mmevk ?= "lmp-device-tree"
-SOTA_CLIENT_FEATURES_append_imx8mmevk = " ubootenv"
```

Comment:
Does this change belong in this PR?  Seems out of place.

Reply:
sorry, should not be here, will drop it